orm/src/test/java/org/hibernate/search/test/embedded/sorting/Villain.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.test.embedded.sorting;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+import org.hibernate.search.annotations.ContainedIn;
+import org.hibernate.search.annotations.Field;
+import org.hibernate.search.annotations.Indexed;
+import org.hibernate.search.annotations.SortableField;
+
+/**
+ * @author Davide D'Alto
+ */
+@Entity
+@Indexed
+class Villain {
+
+	@Id
+	@SortableField
+	@GeneratedValue
+	private Integer id;
+
+	@OneToOne
+	@ContainedIn
+	private Hero hero;
+
+	@Field
+	private String name;
+
+	public Villain() {
+	}
+
+	public Villain(Integer id, String name) {
+		super();
+		this.id = id;
+		this.name = name;
+	}
+
+	public Integer getId() {
```

Comment:
Hi @DavideD 

(working with Yoann on this issue)

The `@SortableField` annotation is not alone. We have the same issue with `@NumericField`. If you add just here:
```java
	@Field(name = "ID_SORT")
	@SortableField(forField = "ID_SORT")
	@NumericField(forField = "ID_SORT")
```
you end up with the following error on startup:
```
org.hibernate.search.exception.SearchException: HSEARCH000262: @NumericField annotation is used on org.hibernate.search.test.embedded.sorting.Villain#id without a matching @Field annotation
	at org.hibernate.search.engine.metadata.impl.NumericFieldsConfiguration.validate(NumericFieldsConfiguration.java:89)
	at org.hibernate.search.engine.metadata.impl.AnnotationMetadataProvider.initializeMemberLevelAnnotations(AnnotationMetadataProvider.java:901)
	at org.hibernate.search.engine.metadata.impl.AnnotationMetadataProvider.initializeClass(AnnotationMetadataProvider.java:480)
	at org.hibernate.search.engine.metadata.impl.AnnotationMetadataProvider.checkForIndexedEmbedded(AnnotationMetadataProvider.java:1616)
	at org.hibernate.search.engine.metadata.impl.AnnotationMetadataProvider.initializeMemberLevelAnnotations(AnnotationMetadataProvider.java:890)
	at org.hibernate.search.engine.metadata.impl.AnnotationMetadataProvider.initializeClass(AnnotationMetadataProvider.java:496)
```

This is due to the fact that `NumericFieldsConfiguration.getNumericFieldAnnotation` is only called if the field is in the path of the embedded entity but the `NumericFieldsConfiguration.validate` method checks for all the numeric fields.

I'm going to push a PR with a fix (but how we do it might be debatable).